common/math/math.go

```diff
@@ -36,16 +40,35 @@ func CalculateFee(amount, fee float64) float64 {
 	return amount * (fee / 100)
 }
 
-// CalculatePercentageGainOrLoss returns the percentage rise over a certain
-// period
-func CalculatePercentageGainOrLoss(priceNow, priceThen float64) float64 {
-	return (priceNow - priceThen) / priceThen * 100
+// PercentageChange returns the percentage change between two numbers, v1 is reference value.
+func PercentageChange(v1, v2 float64) float64 {
+	return (v2 - v1) / v1 * 100
+}
+
+// PercentageDifference returns the percentage difference between two numbers
```

Comment:
This definitely needs to say that it's of the average. All the calculators were at pains to point that out too.
Suggest:
```
// PercentageDifference returns difference between two numbers as a percentage of their average
```

Reply:
[nice](https://github.com/thrasher-corp/gocryptotrader/pull/1617/commits/bdc96463b2f21055eb9765faa400d7d72bc331ac)